media-gfx/openvdb/openvdb-9.0.0-r2.ebuild

```diff
@@ -14,23 +14,23 @@ SRC_URI="https://github.com/AcademySoftwareFoundation/${PN}/archive/v${PV}.tar.g
 LICENSE="MPL-2.0"
 SLOT="0/9"
 KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-IUSE="cpu_flags_x86_avx cpu_flags_x86_sse4_2 +blosc doc numpy python static-libs test utils zlib abi6-compat abi7-compat +abi8-compat"
+IUSE="cpu_flags_x86_avx cpu_flags_x86_sse4_2 +blosc doc +nanovdb numpy python static-libs test utils zlib abi6-compat abi7-compat abi8-compat +abi9-compat"
 RESTRICT="!test? ( test )"
 
 REQUIRED_USE="
 	numpy? ( python )
-	^^ ( abi6-compat abi7-compat abi8-compat )
+	^^ ( abi6-compat abi7-compat abi8-compat abi9-compat )
 	python? ( ${PYTHON_REQUIRED_USE} )
 "
 RDEPEND="
 	>=dev-cpp/tbb-2021.4.0:=
 	dev-libs/boost:=
 	dev-libs/jemalloc:=
 	dev-libs/log4cplus:=
-	>=dev-libs/imath-3.1.4-r2:=
```

Comment:
Keep this to ensure we're using an imath without the bad headers (Imath-3 vs Imath)

Reply:
I must have changed and included these two by mistake. I will revert these two changes.